README.md

```diff
@@ -8,5 +8,44 @@ The tutorials found here are _shared tutorials_ that complement more specific tu
 
 Please check out the [learn](https://scverse.org/learn) page on scverse.org for an overview of all tutorials!
 
+# tutorial registry
+
+The purpose of this document is to formalize the criteria to be eligible for external tutorial inclusion into the scverse tutorials listing (https://scverse.org/scverse-tutorials/).
+
+## Criteria to be eligible for tutorial inclusion
+
+We have the following requirement for a package to be eligible to be considered for tutorial submission/mirroring:
+
+1. The package must be a scverse listed ecosystem package and therefore meet all corresponding requirements.
```

Comment:
Can we link to the requirements/process here?

Reply:
I guess all of them should be core or listed ecosystem packages?